### ISSUE TITLE
Integrate only subset from two datasets and have different prepocess

### ISSUE THREAD

**Issue by yulchen810** [satijalab/seurat#7593]
Hi, Dear Seurat team:
Thank you for your patience and effort.
I want to use the data from published to annotate my data, I plan to integrate two subset datasets from two different resources. 
Here is what I do: 
I subset a cell type (astrocyte) from my integrated snRNA-seq dataset, and do the Normalization, Scaling, RunPCA, ..until FindCluster.
I also download an snRNA-seq dataset from NCBI, and subset the same cell type (astrocyte) from it, then do the Normalization, Scaling, RunPCA, ..until FindCluster. 
The preprocess for the two datasets before subset is different, mainly because the sequence depth is different.
My question is that:
1. the preprocess is different for the two datasets, is that ok?
2. if I want to integrate the two subset dataset, do I need to do batch effect or just follow the steps from https://satijalab.org/seurat/articles/integration_large_datasets.html, which are
```
Create a list of Seurat objects to integrate
Perform normalization, feature selection, and scaling separately for each dataset
Run PCA on each object in the list
Integrate datasets, and proceed with joint analysis
```

Thank you very much!
Best

**Comment by longmanz** [comment]
Hi, 
Generally speaking it is fine to integrate datasets with different preprocesses. Just follow the standard integration procedures at https://satijalab.org/seurat/articles/integration_introduction.html, and you can evaluate your integrated dataset by checking the UMAP. Let us know if any issue arises and we will reopen this issue.

**Comment by yulchen810** [comment]
> Hi, 
> Generally speaking it is fine to integrate datasets with different preprocesses. Just follow the standard integration procedures at https://satijalab.org/seurat/articles/integration_introduction.html, and you can evaluate your integrated dataset by checking the UMAP. Let us know if any issue arises and we will reopen this issue. 

Hi, I did integration of the two subset, and I saw a batch effect between them in the umap graph. 
Do I need to re-analyze the reference dataset as I did in my dataset starting from the fastq file? 
By the way, there are 6 fastq files (GSE143758) that the author used in the paper didn't upload to ncbi, which should be needed in the analysis (7months old hippocampus). If I only use 4 fastq files(original 10 files used in the paper) which will be down-sampled a lot, and the quality control of my dataset is more strict and I'm worried the result could be distinct from the author's result. 
Best
